pkg/ccl/changefeedccl/alter_changefeed_test.go

```diff
@@ -609,6 +609,47 @@ func TestAlterChangefeedAddTargetErrors(t *testing.T) {
 	t.Run(`kafka`, kafkaTest(testFn, feedTestNoTenants))
 }
 
+func TestAlterChangefeedDatabaseQualifiedNames(t *testing.T) {
```

Comment:
As you mess with this, probably a good idea to have test cases like these:

```
CREATE CHANGEFEED FOR movr.drivers, printr.drivers
USE movr; CREATE CHANGEFEED FOR movr.drivers FAMILY f1, drivers FAMILY f2 // refers to the same table twice in two different ways
```

No need to add test cases where we haven't defined the right behavior yet, but